### DISCUSSION TITLE
How to transform txt or spc to rc2

### DISCUSSION THREAD

**Discussion by nicocopez** [h2020charisma/oranchada#26] [Q&A] [answered]
Hi all,
Miguel was asking how to convert from txt or spc files to rc2 in  Oranchada. Could you help with this?

**Comment by vedina** [comment, marked as answer]
Hi Nico,

There is no need to convert files . Oranchada can load txt and spc files
using the Load File widget.

Please note there is a new release today, do update oranchada via the
Add-on menu.

Regards,
NIna
